packages/plugin-ext/src/plugin/type-converters.spec.ts

```diff
@@ -437,4 +438,24 @@ describe('Type converters:', () => {
             assert.deepStrictEqual(result, showOptions);
         });
     });
+
+    describe.only('#convertCode', () => {
```

Comment:
`.only` should be removed:

```suggestion
    describe('#convertCode', () => {
```